cli/src/package.rs

```diff
@@ -0,0 +1,58 @@
+extern crate tar;
+extern crate ignore;
+use std::path::Path;
```

Comment:
holy crap. so many uses.

Reply:
yeah a lot of glue to make this happen :)